backend/internal/model/game.go

```diff
@@ -29,11 +33,18 @@ type GameScore struct {
 	WhiteScore int `json:"whiteScore"`
 }
 
+type ShotsData struct {
+	WhiteCount int
+	BlueCount  int
+	Fastest    adapter.ShotMessage
```

Comment:
I don't think model should be aware of the adapter

Reply:
Should we make another structure in game.go? 
We thought it's a bad Idea because it would be identical to the one from adapter

Reply:
> > tests for new game functionality?
> 
> We didn't want to add them here to not make the PR even bigger, we added them in draft #99

Your DOD specifies that tests are required. Tests should come with the functionality they are testing, to prove that it's working as intended.

Reply:
Using struct from the adapter defeats the purpose of the adapter

Reply:
We moved tests to this PR and closed the draft

Reply:
> Using struct from the adapter defeats the purpose of the adapter

Should we have the structure in model and use it in adapter, or have two structures?

Reply:
The purpose of the adapter is to separate structs dependent on external services (eg service detecting shots) from internal structs, used by the backend. You should have all the structs needed for business logic defined in `model` in the cleanest and most handy way possible, and then in the adapter, you should do what's required to convert from external structures to internal structures.

Reply:
We moved the struct to `model`